pkgs/development/rocm-modules/5/clr/default.nix

```diff
@@ -82,7 +82,7 @@ in stdenv.mkDerivation (finalAttrs: {
     "-DROCM_PATH=${rocminfo}"
 
     # Temporarily set variables to work around upstream CMakeLists issue
-    # Can be removed once https://github.com/ROCm-Developer-Tools/hipamd/issues/55 is fixed
+    # Can be removed once https://github.com/ROCm/hipamd/issues/55 is fixed
```

Comment:
This seems to be still the old link

Reply:
Changed to https://github.com/ROCm/rocm-cmake/issues/121.